docs/conf.py

```diff
@@ -42,7 +42,7 @@
     "sphinx.ext.mathjax",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
```

Comment:
Shouldn't this require `nbsphinx`? https://github.com/projectmesa/mesa/blob/d174b9ca7dcaa9c95c74d03ba6b4aaca227aac12/docs/conf.py#L45-L46.
See https://github.com/projectmesa/mesa/commit/9b4ba3aea74060c4c100aed7f4734b55a8b3f2c3 -- you have to add .ipynb in https://github.com/projectmesa/mesa-geo/blob/4c1a039efafa0bf6392284d9e66b5949aa628de8/docs/index.md?plain=1#L40.

Reply:
Wait, that's an external link, so doesn't require modification. But `nbsphinx` is probably still necessary.

Reply:
This line
https://github.com/projectmesa/mesa-geo/blob/4c1a039efafa0bf6392284d9e66b5949aa628de8/docs/index.md?plain=1#L71
needs .ipynb.

Reply:
From this [link](https://docs.readthedocs.io/en/stable/guides/jupyter.html), one of `nbsphinx`/`myst_nb` needs to be enabled in the extensions list. Also in their example, the `.ipynb` file extension is not necessary, although it may still work with the extension.

Reply:
Tested on https://github.com/projectmesa/mesa/pull/1754. It worked.